components/NaveSlider.vue

```diff
@@ -124,12 +122,7 @@ export default {
     },
 
     getPosition () {
-      let offsetTrail = this.$refs.sliderContent.lastChild
-      let offsetLeft = 0
-      while (offsetTrail) {
-        offsetLeft += offsetTrail.offsetLeft
-        offsetTrail = offsetTrail.offsetParent
-      }
+      const offsetLeft = this.$refs.sliderContent.lastChild.offsetLeft
 
       this.position = offsetLeft
```

Comment:
this.position = this.$refs.sliderContent.lastChild.offsetLeft

Reply:
aaaaaaaann